backend-node/routes/vonage.js

```diff
@@ -29,25 +18,6 @@ const fromHeaderOrQuerystring = (req) => {
 
 const uploadImage = multer({
```

Comment:
I think this should be removed, right?
```suggestion
const uploadImage = multer({
     storage: multerS3({
```

Reply:
thanks